libflux/flux-core/src/formatter/mod.rs

```diff
@@ -516,16 +519,33 @@ impl Formatter {
 
     fn format_function_expression(&mut self, n: &ast::FunctionExpr) {
         self.format_comments(&n.lparen);
+        let multiline = n.params.len() > 4 && n.base.is_multiline();
```

Comment:
4 seems like a good minimum to change this. Have we considered func signature length as well or was that idea rejected?

Reply:
I'm not sure, I think 4 is standard throughout the stdlib, we just emulated it.